src/main/java/mate/hwdao/controllers/AddCarController.java

```diff
@@ -0,0 +1,42 @@
+package mate.hwdao.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.hwdao.Main;
+import mate.hwdao.lib.Injector;
+import mate.hwdao.model.Car;
+import mate.hwdao.model.Manufacturer;
+import mate.hwdao.service.CarService;
+import mate.hwdao.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+
```

Comment:
Don’t begin class or method implementation with an empty line

Reply:
https://google.github.io/styleguide/javaguide.html#s4.6-whitespace

Reply:
I think it makes some sense as it became a `common mistake`. But on another hand it makes sense as written here :
"A blank line before the first member or initializer, or after the last member or initializer of the class, is neither encouraged nor discouraged."